internal/ip/database/example_test.go

```diff
@@ -0,0 +1,59 @@
+package database_test
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/brexhq/substation/internal/ip/database"
+)
+
+func Example_iP2Location() {
+	// the location of the enrichment database can be either a path on local disk, an HTTP(S) URL, or an AWS S3 URL
+	location := "location://path/to/ip2location.bin"
+
+	// create IP2Location container, open database, and close database when function returns
+	ip2loc := database.IP2Location{}
+	if err := ip2loc.Open(context.TODO(), location); err != nil {
+		// handle error
+		panic(err)
+	}
+	defer ip2loc.Close()
+
+	// query database
+	addr := "8.8.8.8"
```

Comment:
docs: can the test IPs be pulled from RFC5737 (IPv4 Address Blocks Reserved for Documentation) like using `192.0.2.1`?

Reply:
we can, but they produce no output. for example, this is the output for that IP address: `{"location":{"coordinates":{}}}`

Reply:
Ah, bummer that these down't have output but make sense. Better to leave one with data I guess.